### PR TITLE
create index.html and sketch.js

### LINKED ISSUE #1: Create p5.js sketch
When the command is first run it should create `index.html` and `sketch.js`.

### PR DESCRIPTION
**Pull request by JuggernautJf**
added index.html and sketch.js using setupFiles command
fixes #1

### CODE REVIEW

File: src/commands/setupLayout.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ class SetupLayoutCommand extends Command {
     }
 
     async run() {
-        // TODO: CREATE index.html and sketch.js
```
Comment (JuggernautJf) [inline src/commands/setupLayout.js:12]: Done

**Comment by supercrafter100** [comment]
Please see the review comments on your other PR, they don't appear to be addressed here.

**Comment by JuggernautJf** [comment]
https://github.com/CodingTrain/Bizarro-Devin/pull/16#pullrequestreview-1965153067

**Comment by JuggernautJf** [comment]
**done** and *done*

**Review by supercrafter100 — APPROVED** [review]

**Comment by JuggernautJf** [comment]
@shiffman think its good to go then? 👉👈

**Comment by shiffman** [comment]
Wonderful, thank you @JuggernautJf I'm going to merge! I'll be working on this project a lot of over the weekend so I would suggest not making too many significant changes without opening an issue for discussion first, thank you for your generous spirit!

**Pull request merged**